helm_deploy/nomis-prisoner-deletion-service/values.yaml

```diff
@@ -13,11 +13,8 @@ generic-service:
     annotations:
       cert-manager.io/cluster-issuer: letsencrypt-production
     enabled: true
-    v1_2_enabled: true
```

Comment:
Defaults in helm config are `v1_2_enabled` set to `false` and `v0_47_enabled` set to `true`, is that what you want?